File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        
        int discount = (price >= 500000) ? 20 : (price >= 300000) ? 10 : (price >= 100000) ? 5 : 0;
        return price * (100 - discount) / 100;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.js
function solution(price) {
    let discount = 0;

    if (price >= 500000) {
        discount = 20;
    } else if (price >= 300000) {
        discount = 10;
    } else if (price >= 100000) {
        discount = 5;
    }

    const finalPrice = price * (1 - discount / 100);
    return Math.floor(finalPrice); 
}